From 75d6eaaac7aecb32cf0c2de229335fc257251499 Mon Sep 17 00:00:00 2001
Message-ID: <75d6eaaac7aecb32cf0c2de229335fc257251499.1757421429.git.mjg@fedoraproject.org>
From: Michael J Gruber <mjg@fedoraproject.org>
Date: Tue, 9 Sep 2025 14:36:17 +0200
Subject: [PATCH] mupdfwrap_test: adjust to new filter arguments

2b126ac3e ("Add PDF_TEXT_REMOVE_INVISIBLE.", 2025-09-05)
in mupdf 1.27.x added new filter arguments. Adjust to this.
---
 scripts/mupdfwrap_test.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/mupdfwrap_test.py b/scripts/mupdfwrap_test.py
index 6b16fdc63..a7a4e2ef2 100755
--- a/scripts/mupdfwrap_test.py
+++ b/scripts/mupdfwrap_test.py
@@ -86,7 +86,7 @@ def test_filter(path):
             super().__init__()
             self.use_virtual_text_filter()
             self.state = 1
-        def text_filter( self, ctx, ucsbuf, ucslen, trm, ctm, bbox):
+        def text_filter( self, ctx, ucsbuf, ucslen, trm, ctm, bbox, tr, ca, CA):
             if 0:
                 log( f'text_filter(): ctx={ctx} ucsbuf={ucsbuf} ucslen={ucslen} trm={trm} ctm={ctm} bbox={bbox}')
             # Remove every other item.
-- 
2.51.0.432.gc8867dcbdc

